### DISCUSSION TITLE
Should there be an option to cancel the HIBP checks?

### DISCUSSION THREAD

**Discussion by kapsiR** [kapsiR/HaveIBeenPwnedKeePassPlugin#9] [Ideas]
Should there be any support for cancellation? 

1. How should this look like?
Cancel the current operation via the global menu?

2. How should parallel tasks be handled? (multiple open databases)

**Comment by ghost** [comment]
i think it would be nice for people with little internet connexion to show cancellation if data don't respond fast enough
